wgpu-core/src/lib.rs

```diff
@@ -67,7 +67,11 @@ pub mod registry;
 pub mod resource;
 pub mod storage;
 mod track;
-mod validation;
+// This is public for users who pre-compile shaders while still wanting to
+// preserve all run-time checks that `wgpu-hal` does.
+// See <https://github.com/gfx-rs/wgpu/issues/3103>, after which this can be
+// made private again.
```

Comment:
Let me know if this can be improved, unfortunately English isn't my first language.

Reply:
One small typo, otherwise good!